components/Workspaces/WorkspaceWelcomeModal.tsx

```diff
@@ -43,11 +43,17 @@ export default function WorkspaceWelcomeModal({ isOpen, onClose }: WorkspaceWelc
       title={TITLE}
       description={
         <div className="text-center flex flex-col items-center justify-between gap-4 !p-8">
-          <img
-            src="/assets/workspace_overview.png"
-            alt="Workspace screenshot from documentation"
+          <iframe
+            width="100%"
+            height="300px"
             className="border-2 border-light-orange-9 shadow-md rounded-lg mb-4"
-          />
+            src="https://www.youtube-nocookie.com/embed/xpxIyR4-rRc?si=UwT2Ukmjc08hFjU-"
+            title="Welcome to Workspaces"
+            frameBorder="0"
```

Comment:
frameborder is deprecated.

![CleanShot 2024-05-21 at 11 07 17](https://github.com/open-sauced/app/assets/833231/5e5026ce-3684-443c-a7a5-1c0363e225cc)

```suggestion
```